src/test/java/com/futurecollars/lesson11/zad6/ResultWorkoutTest.java

```diff
@@ -0,0 +1,20 @@
+package com.futurecollars.lesson11.zad6;
+
+import com.futurcollars.lesson11.zad6.ResultWorkout;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+
+public class ResultWorkoutTest {
+
+    @ParameterizedTest
+    @CsvSource(value = {"3:Doskonała"}, delimiter = ':')
+    void shouldGetResultWorkout(int input, String expected) {
+        // given
+        ResultWorkout resultWorkout = new ResultWorkout(61, 400, 400);
```

Comment:
Wogóle w teście nie używasz zmiennej input, więc nie jest to test parametryczny. Dane o długości treningu, kaloriach też powinny być podane jako parametry.